module/manage_requests.py

```diff
@@ -0,0 +1,75 @@
+import yaml
+from telegram import InlineKeyboardButton, InlineKeyboardMarkup
+from telegram.ext import CallbackContext
+from module.create_connection import create_connection
+from module.add_item import add_item
+from module.find import find
+from module.shared import DB_PATH, error_message
+
+
+def add_request(context: CallbackContext, chat_id: int, isbn: str, title: str, authors: str, username: str, price: str) -> None:
+    item = (chat_id, isbn, title, authors, username, price)
+    sql = """ INSERT INTO Requests(ChatID, ISBN, Title, Authors, Seller, Price)
+              VALUES(?,?,?,?,?,?) """
+    conn = create_connection(DB_PATH)
+    if not conn:
+        context.bot.send_message(chat_id, error_message)
+        return
+    cur = conn.cursor()
+    cur.execute(sql, item)
+    conn.commit()
+    conn.close()
+    return cur.lastrowid
+
+def delete_request(context: CallbackContext, chat_id: int, row_id: int) -> None:
+    conn = create_connection(DB_PATH)
+    if not conn:
+        context.bot.send_message(chat_id, error_message)
+        return
+    cur = conn.cursor()
+    cur.execute("DELETE FROM Requests WHERE rowid=?", (row_id,))
+    conn.commit()
+    conn.close()
+
+def send_request(context: CallbackContext, row_id: int) -> None:
+    with open('config/settings.yaml', 'r') as yaml_config:
+            config_map = yaml.load(yaml_config, Loader=yaml.SafeLoader)
+    group_id = config_map['admin_group_id']
+    keyboard = [[InlineKeyboardButton('Y', callback_data=('new_request;Y;' + str(row_id)))], [InlineKeyboardButton('N', callback_data=('new_request;N;'+ str(row_id)))]]
```

Comment:
this `'Y'` and `'N'` need to be in constants variables shared in the project ref https://testdriven.io/blog/clean-code-python/#6-dont-use-magic-numbers